scripts/agent-hooks/git-guard.sh

```
@@ -86,12 +86,36 @@ esac
+        rest=${rest#=} ;;
+    esac
+    case "$cmd" in
+      *" -m"*)
+        m_rest=${cmd#*" -m"}
+        [ ${#m_rest} -gt ${#rest} ] && rest=$m_rest ;;
+    esac
+    # First line of the remainder tells us the message form. An empty result
+    # means no inline message (editor/--amend/-F) — left alone.
+    first=$(printf '%s\n' "$rest" | sed -n '1s/^[[:space:]]*//p')
+    case "$first" in
+      *'$('*'<<'*)
```

Shellcheck (0.11.0)

[info] Expressions don't expand in single quotes, use double quotes for that.

(SC2016)